src/utils/getWeightDivID.js

```diff
@@ -0,0 +1,41 @@
+const csv = require("csv-parser");
+const fs = require("fs");
+const path = require("path");
+
+const getWeightDivID = async (sex, weight, age) => {
+  // Dynamically find the path to the age_div.csv file (workaround for Jest testing environment)
+  let weightDivPath = __dirname;
+  while (!weightDivPath.endsWith("src")) {
+    weightDivPath = path.resolve(weightDivPath, "..");
+  }
+  weightDivPath = path.resolve(
```

Comment:
i think the while loop isn't really ideal here. we should find a way to add in the filepath as one string

maybe this might work?:
`path.resole("@/app/sql/database/weight_div.csv")`

Reply:
yeaaa I rlly didn't want to use the loop either. I tried using the path.resolve("@/app/sql/database/weight_div.csv"), path.resolve("@src/app/sql/database/weight_div.csv"), path.resolve("@/src/app/sql/database/weight_div.csv"), but none worked. :(